integration/hostuser_test.go

```diff
@@ -513,6 +513,41 @@ func TestRootHostUsers(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, expectedShell, userShells[namedShellUser])
 	})
+
+	t.Run("Test expiration removal", func(t *testing.T) {
```

Comment:
I don't think there's a very good way to test the creation case without exposing a way to create a user with expirations set before the removal function is called. I added an `Expired` field to `HostUsersInfo` and `HostUserOpts` to facilitate this which I'm not a huge fan of since it only exists for testing purposes.

The only other option would be to fake both a `HostUsers` and `HostUsersBackend` that calls `useradd` with alternate flags which, now that I'm typing this, probably wouldn't be quite as much work as I originally thought. I'll spend a few minutes putting together some fakes and if it comes together easily I'll remove the `Expired` fields.

Reply:
Building out the fake ended being fairly straightforward, so no more "useless" `Expired` fields :tada: